CodingChallenges/CC_140_Pi_Leibniz/Processing/CC_140_Pi_Leibniz/CC_140_Pi_Leibniz.pde

```diff
@@ -0,0 +1,43 @@
+// Pi Day Leibniz Series
+// Daniel Shiffman
+// https://thecodingtrain.com/CodingChallenges/140-leibniz-formula-pi
+// https://editor.p5js.org/codingtrain/sketches/8nvCqk0-W
+// https://youtu.be/uH4trBNn540
+
+float pi = 4;
+int iterations = 0;
+ArrayList<Float> history = new ArrayList<Float>();
```

Comment:
Would you mind using Processing's `FloatList` instead? 

https://processing.org/reference/FloatList.html

Reply:
Sure!